app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MessagesController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  skip_before_action :authenticate_user!, :only => "sms"
```

Comment:
Is there any identifier from the twilio request that we could use to help us prevent requests from other sources?